### ISSUE TITLE
Message is show after the reload the page

### ISSUE THREAD

**Issue by CoderRathore** [nahid/talk#75]
i have implement the chat functionality in project but when i send message to another user than message will not show but when we reload the page the message will show correct. so i want to solve this problem pls help me.

**Comment by CoderRathore** [comment]
Pls help me sir....

**Comment by gaurav1992** [comment]
@nahid  Could you please respond on the same. It is really necessary if we are using this library.

**Comment by FrenchMajesty** [comment]
Same deal here. It seems messages are not being sent to Pusher.

**Comment by cristianuibar** [comment]
This is happening because of: `Call to undefined method Vinkla\\Pusher\\Facades\\Pusher::trigger()`

**Comment by cristianuibar** [comment]
Ok, found the problem. In Broadcast.php you should use:
```
use Pusher\Pusher;
```
instead of 
```
use Pusher;
```

Thats it. And also make sure you use the right Cluster in your config.

**Comment by nahid** [comment]
You have to update Talk with version `v2.2.1` and also update Talk config file. The config file is contains cluster info so now you can setup cluster by this config.

**Comment by cristianuibar** [comment]
Yes, I did that. Yet until I added Pusher/Pusher in Broadcast.php I was not able to get the Pusher to work. Now it's working fine and I had to forcefully add that file to my git tracking and whitelist it in gitignore. Not elegant but did the trick.

**Comment by nahid** [comment]
Please make a PR for this update

**Comment by tobidsn** [comment]
Same deal here, 
I am using Laravel 5.3 , nahid/talk = 2.2.1
and Server Centos 7

**Comment by tobidsn** [comment]
SOLVED for me, 

// old controller method
$conversations = Talk::getConversationsById($conversationId);

Add Take parameter in getConversationsById
// new controller method
$conversations = Talk::getConversationsByUserId($id, $offset = 0, $take = 1000);

$messages = $conversations->messages;
$withUser = $conversations->withUser;
return view('messages.conversations', compact('messages', 'withUser'));

hi @nahid whether "take" can be use for "Paginate" ?

**Comment by nahid** [comment]
@tobidsn yes take is used for pagination.

**Comment by rvkvino** [comment]
Hello,
I have downloaded and checked this latest version talk example. And I have created account in pusher and I have configured in this .env file also. But still not it's working in live update. I can get the messages if I reload the page. Live update not happening in page. Could you please help me on this ASAP.
